Remotion/Web/Development.WebTesting/WebDriver/Configuration/Firefox/FirefoxBinariesProvider.cs

```diff
@@ -88,17 +88,26 @@ private string GetDriverPathAndDownloadIfMissing (string firefoxPath)
                 s_minimumSupportedFirefoxVersion.ToString (1)));
       }
 
-      var driverReleaseInfo = GetLatestGeckoDriverReleaseInfo();
+      var initialSecurityProtocol = ServicePointManager.SecurityProtocol;
+      try
+      {
+        ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Please try to change the compatibility setting in the `app.config` instead of changing the values of `ServicePointManager.SecurityProtocol`.